lib/internal/inspector_network_tracking.js

```diff
@@ -22,18 +22,26 @@ function onClientRequestStart({ request }) {
     request: {
       url,
       method: request.method,
+      headers: request.getHeaders(),
     },
   });
 }
 
-function onClientResponseFinish({ request }) {
+function onClientResponseFinish({ request, response }) {
   if (typeof request._inspectorRequestId !== 'string') {
     return;
   }
+  const url = `${request.protocol}//${request.host}${request.path}`;
   const timestamp = DateNow() / 1000;
   Network.responseReceived({
     requestId: request._inspectorRequestId,
     timestamp,
+    type: 'Other',
+    response: {
+      url,
+      status: response.statusCode,
```

Comment:
`statusText` is also a required field: https://chromedevtools.github.io/devtools-protocol/tot/Network/#type-Response.

```suggestion
      status: response.statusCode,
      statusText: response.statusMessage,
```